website/snippets/_new-sl-setup.md

```diff
@@ -0,0 +1,33 @@
+You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+- You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must have a successful run in your new environment.
+
+:::tip
+If you're using the legacy Semantic Layer, we **highly** recommend you [upgrade your dbt version](/docs/dbt-versions/upgrade-core-in-cloud) to dbt v1.6 or higher to use the new dbt Semantic Layer. Refer to the dedicated [migration guide](/guides/migration/sl-migration) for more info.
+:::
+
+1. in dbt Cloud, create a new [deployment environment](/docs/deploy/deploy-environments#create-a-deployment-environment) or use an existing environment on dbt 1.6 or higher.
+    * Note &mdash; Deployment environment is currently supported (_development experience coming soon_)
+
+2. Navigate to **Account Settings** and select the specific project you want to enable the Semantic Layer for.
+
+3. In the **Project Details** page, navigate to the **Semantic Layer** section, and select **Configure Semantic Layer**.
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-initial-page.jpg" width="60%" title="Semantic Layer section in the Project Details page"/>
+
+4. In the **Set Up Semantic Layer Configuration** page, enter the credentials you want the Semantic Layer to use specific to your data platform. 
+    * Note: We recommend using a less privileged set of credentials because your Semantic Layer users will be querying it in downstream applications. At a minimum, the Semantic Layer needs to have read access to the schema(s) that contains the dbt models that you used to build your semantic models.
```

Comment:
I think a note doesn't need to also be a bullet. Also, I rephrased "privilege set" for those less familiar

```suggestion
**Note:** We recommend credentials have the least privileges required because your Semantic Layer users will be querying it in downstream applications. At a minimum, the Semantic Layer needs to have read access to the schema(s) that contains the dbt models that you used to build your semantic models.
```